app/src/main/java/com/atomicrobot/carbon/ui/main/MainViewModel.kt

```diff
@@ -54,6 +54,7 @@ class MainViewModel(
 
             when (value) {
                 is Commits.Error -> snackbarMessage.value = value.message
+                else -> {}
             }
```

Comment:
I think it would better to include the other branches because it forces you to consider functionality if a new class is added to the sealed class. @atomicrobotjohn do you have any strong opinions either way?

Reply:
I prefer to include the other branches over a generic else as well.

Reply:
Replaced the default else with all possible branches.